server/src/main/java/org/cloudfoundry/identity/uaa/oauth/UaaTokenStore.java

```diff
@@ -100,7 +100,8 @@ public String createAuthorizationCode(OAuth2Authentication authentication) {
         performExpirationClean();
         JdbcTemplate template = new JdbcTemplate(dataSource);
         int tries = 0;
-        while ((tries++)<=max_tries) {
+        while (true) {
+            tries++;
```

Comment:
Interesting. I'm able to comment on a line of code in an individual commit. Here I'm seeing the name of the variable as "tries", which was later refactored.